docs/usage.md

```diff
@@ -114,20 +114,20 @@ npm install --save-dev @babel/preset-env
 ./node_modules/.bin/babel src --out-dir lib --presets=@babel/env
 ```
 
-Without any configuration this preset will include all the es2015, es2016 and es2017 plugins. But presets can take options too. Rather than passing both cli and preset options form the terminal, let's look at another way of passing options: configuration files.
+Without any configuration this preset will include all the es2015, es2016 and es2017 plugins. But presets can take options too. Rather than passing both cli and preset options from the terminal, let's look at another way of passing options: configuration files.
 
 ## Configuration
 
-There exists multiple type of configuration in Babel but we will use the `babel.config.js` one. You can read more about configuration here: [TODO CONFIGURATION SECTION FIRST](https://oo).
+You can read more about how to [Configure Babel](configuration.md) next. For now we will create a file called `babel.config.js` with this content:
```

Comment:
Thoughts on adding a little bit more to the first bit?

---

### Configuration

> There are a few different ways to use configuration files depending on your needs. Be sure to read our in-depth guide on how to [configure Babel](configuration.md) for more information.

For now, let's create a file called `babel.config.js` with the following content:

```
<insert example>
```